eng/common/templates/job/onelocbuild.yml

```diff
@@ -22,13 +22,14 @@ parameters:
   MirrorRepo: ''
   MirrorBranch: main
   condition: ''
+  JobNameSuffix: ''
 
 jobs:
-- job: OneLocBuild
+- job: OneLocBuild${{ parameters.JobNameSuffix }}
```

Comment:
unfortunately this likely won't work. Next arcade update likely overwrites this change.

Reply:
it's now part of arcade code: https://github.com/dotnet/arcade/blob/main/eng/common/templates/job/onelocbuild.yml#L30